imap_processing/tests/test_utils.py

```diff
@@ -7,6 +7,10 @@
 
 from imap_processing import utils
 
+pytest_plugins = [
+    "imap_processing.tests.swapi.test_swapi_l1",
+]
```

Comment:
@subagonsouth, I used your plugins idea here to bring in an external semi-unrelated fixture that probably shouldn't be brought up into the main `conftest.py` level. Does this make sense to you here, or is there a different/better way to bring in external fixtures? Maybe you have an even better way of addressing the test coverage here though.

Reply:
So, this apparently didn't actually work the way I thought it did... It did work locally for me, but not remotely. Reading more about it on pytest it looks like it may not be supported to redefine `pytest_plugins` outside of the root `conftest.py`?

I ended up calling the function directly now instead of the fixture, which also allowed me to parametrize the test. Still curious about your thoughts on making this better though.